### DISCUSSION TITLE
Allow a simple 'unnumbered: True' attribute on links

### DISCUSSION THREAD

**Discussion by jbemmel** [ipspace/netlab#91] [Ideas]
https://github.com/ipspace/netsim-tools/blob/master/docs/addressing.md#unnumbered-interface-support works, but is more complicated than it needs to be imho

Instead of:
```
role: <some-custom-pool-with-unnumbered-flag-set>
```

why not simply:
```
unnumbered: True
```

**Comment by ipspace** [comment]
I wanted to have a mechanism where I could do network-wide change from numbered to unnumbered (use case was OSPF unnumbered), but of course we can add the **unnumbered** flag and a bit of logic around address allocation. Will do.

**Reply by jbemmel** [reply]
I can see the reasoning to support large scale networks, but sometimes people simply want to focus on a single link. Having both options would be great, thanks

**Comment by ipspace** [comment]
Implemented in #92 / #97
